Library/Homebrew/download_strategy.rb

```diff
@@ -458,7 +458,7 @@ def resolve_url_basename_time_file_size(url)
         filename = URI.decode_www_form_component(encoded_filename).encode(encoding) if encoding && encoded_filename
       end
 
-      filename || content_disposition.filename
+      (filename || content_disposition.filename).rpartition("/")[-1]
```

Comment:
@Bo98 Can you add a comment here? It's not clear what this is doing or why.

Reply:
Yes, will do.